src/utils/isValidAddress.ts

```diff
@@ -0,0 +1,11 @@
+import { isAddress, isHexStrict } from 'web3-utils'
+
+export const isValidAddress = (address?: string): boolean => {
+  if (address) {
+    // `isAddress` do not require the string to start with `0x`
+    // `isHexStrict` forces the address to start with `0x`
+    return isAddress(address) && isHexStrict(address)
```

Comment:
`isHexStrict` should probably go first because `isAddress` potentially runs a checksum.

Reply:
no, it's Ok. `isAddress` won't throw for an invalid address. But you make me realize that I forgot a couple of cases in the tests [`a3d940b` (#2355)](https://github.com/gnosis/safe-react/pull/2355/commits/a3d940b7f6d8518dbf9c138f496d2ba1084367fd).

Reply:
I meant in terms of performance. First a simple string check, then a more complicated checksum check. I know it won't throw but according to the docs it might check the sum internally.

Reply:
oh, <img src="https://user-images.githubusercontent.com/3315606/120229996-c682bc80-c224-11eb-9c84-1ce6993ad643.png" width="24" />

Done, thanks!